### ISSUE TITLE
which is not running for snaps on core22 base

### ISSUE THREAD

**Issue by alfonsosanchezbeato** [snapcore/core-base#70]
A permission denied happens as it is a link to `/usr/bin/which.debianutils`, which is not granted by apparmor rules (see `template.go`).

**Comment by alfonsosanchezbeato** [comment]
https://github.com/snapcore/snapd/pull/11938

**Comment by Meulengracht** [comment]
This was released under snapd 2.56, and I verified this can now run on uc22:

```
the-meulengracht@ubuntu:~$ which grep
/usr/bin/grep
the-meulengracht@ubuntu:~$ snap list
Name                 Version                 Rev    Tracking       Publisher         Notes
core18               20220706                2538   latest/stable  canonical✓        base
core22               20220706                275    latest/edge    canonical✓        base
pc                   22-0.3                  128    22/edge        canonical✓        gadget
realtime-kernel      5.15.0-1016.16.1        x1     -              -                 kernel
realtime-tests-phme  2.4                     9      latest/stable  the-meulengracht  -
snapd                2.56.3+git761.g293c48d  16695  latest/edge    canonical✓        snapd
the-meulengracht@ubuntu:~$ 
```

**Comment by Meulengracht** [comment]
This was released under snapd 2.56, can we verify this is now working, or is there a specific snap I can use to this test?

**Comment by alfonsosanchezbeato** [comment]
I did a quick test and is working with 2.56.2.
